### PR TITLE
[10.0][FIX] Update README.md

### PR DESCRIPTION
**Pull request by elicoidal**
Added Badges
Added introduction

### CODE REVIEW

**Comment by elicoidal** [comment]
I am not sure about coverall and how to get the list of addons updated in the README

**Comment by pedrobaeza** [comment]
Coveralls is not used anymore. Use Codecov badge.

**Comment by elicoidal** [comment]
@pedrobaeza

**Comment by astirpe** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# business-requirement
-Business Requirements Management via Odoo
+[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/222/10.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-business-requirement-222)
+[![Build Status](https://travis-ci.org/OCA/business-requirement.svg?branch=10.0)](https://travis-ci.org/OCA/business-requirement)
+[![codecov](https://codecov.io/gh/OCA/business-requirement/branch/10.0/graph/badge.svg)](https://codecov.io/gh/OCA/project)
```
Comment (astirpe) [inline README.md]: Must be `business-requirement` instead of `project` . Please check

**Review by astirpe — CHANGES_REQUESTED** [review]

**Comment by elicoidal** [comment]
@astirpe

**Review by astirpe — APPROVED** [review]

**Review by sbidoul — CHANGES_REQUESTED** [review]
The addons placeholder is incorrect, see example in #75

**Review by dreispt — APPROVED** [review]
To squash merge

**Pull request merged**